app/models/hydrus/collection.rb

```diff
@@ -93,7 +93,8 @@ def items_list(opts = {})
     items = []
     items_from_solr.each do |solr_doc|
       id = solr_doc['id']
-      title = self.class.object_title(solr_doc)
+      solr_title = self.class.object_title(solr_doc)
+      title = solr_title.blank? ? 'Untitled' : solr_title
```

Comment:
object_title already returns `Untitled` see #300

Reply:
something went wrong when creating the data then:

```
coll=Hydrus::Collection.find('druid:md919gh6774');nil;
@items = coll.items_list;nil;
@items.each {|i| puts "#{i[:pid]} : #{i[:title]} : #{i[:title].blank?}"};nil;
```

Returns one items like this:

```
druid:fz270ww2247 :  : true
```

Reply:
and in solr:

`https://sul-solr-a.stanford.edu/solr/hydrus_prod_blue/select?q=id%3A%22druid%3Afz270ww2247%22`

```
"titleInfo_title_ssm":["",
          ""],
"titleInfo_0_title_ssm":["",
          ""]
```